installation/installing-images/README.md

```diff
@@ -40,3 +40,12 @@ For more advanced control of this process, see our system-specific guides:
 - [Mac OS](mac.md)
 - [Windows](windows.md)
 - [Chrome OS](chromeos.md)
+
+
+## Boot your new OS
+
+You can now insert the SD card into the Raspberry Pi and power it up. 
+
+For the official Raspbian OS, if you need to enter it, the default user name is `pi`, with password `raspberry`. Remember the default keyboard layout is set to UK.
```

Comment:
Perhaps "if you need to enter it" should be replaced with "if you need to manually login" ? (IIRC most new installs just boot straight to a logged-in desktop?)